input/pagecontent/index.md

```diff
@@ -115,14 +115,14 @@ This guide is divided into several pages which are listed at the top of each pag
 - [Guidance](guidance.html): These pages list the guidance for this guide.
   - [General Guidance](general-guidance.html): This page provides guidance on using the profiles defined in this guide.
   - [Medicine Information](medicine-information.html): This page provides guidance on constructing medications related resources.
-  - [AU Core Data for Interoperability](aucdi.html): This page maps AUCDI data groups and elements to FHIR artefacts in AU Core.
+  - [AU Core Data for Interoperability](aucdi.html): This page provides guidance on the relationship between AU Core, AUCDI, and other implementation guides.
```

Comment:
This change, was applied to the wrong line. Put back the line for AU Core Data for Interoperability and move this new text to the line for Relationship with Other IGs.